fuse/plugins/detector_physics/s2_photon_propagation.py

```diff
@@ -583,7 +564,6 @@ def s2_pattern_map_diffuse(self, n_electron, z, xy, drift_time_mean):
             diffusion_constant_radial *= 1e-9  # cm²/ns
             diffusion_constant_azimuthal *= 1e-9  # cm²/ns
         else:
-            # diffusion_constant_transverse = diffusion_constant_transverse
             diffusion_constant_radial = self.diffusion_constant_transverse
             diffusion_constant_azimuthal = self.diffusion_constant_transverse
```

Comment:
Do we need a separate configuration for azimuthal diffusion constant?

Reply:
Also, `enable_diffusion_transverse_map` actually means whether to use a map for both transverse and azimuthal diffusion constant, so this name is a bit misleading for me.

Reply:
When we do not use the map, radial and azimuthal both become the generic transverse diffusion coefficient, hence I would say this is correct.

Reply:
Before it was tricky to make the difference, but I now added the `enable_` for the booleans that decide if we call the map or not.